mne/preprocessing/xdawn.py

```diff
@@ -0,0 +1,566 @@
+"""Xdawn implementation."""
+# Authors: Alexandre Barachant <alexandre.barachant@gmail.com>
+#
+# License: BSD (3-clause)
+
+
+import numpy as np
+from scipy import linalg
+import copy as cp
+from distutils.version import LooseVersion
```

Comment:
import order should be: 1) standard library 2) numpy scipy / other libraries 3) our code -- so move copy up.